dlrs/main/classes/RollupService.cls

```diff
@@ -1059,7 +1059,8 @@ global with sharing class RollupService {
             '[\r\n]+'
           )
         ) {
-          lookupFields.add(criteriaField);
+          String trimmedCriteriaField = criteriaField.trim();
+          lookupFields.add(trimmedCriteriaField);
```

Comment:
Does this need to be staged into a string or could you `trim` the thing you're adding directly?